docs/Administrator-Guide/Events-APIs.md

```diff
@@ -320,32 +326,32 @@ BRICK_REPLACE                  | volume               | Volume Name
 Event Type                     | Attribute            | Description
 ------------------------------ | -------------------- | -----------
 GEOREP_CREATE                  | force                | Force option used during session Create
-                               | secondary            | Secondary Details(Secondaryhost::SecondaryVolume)
```

Comment:
Why these changes are made here and in other places as well?

Reply:
Strange,,,I think the OR needs to be changed :/ I think my remote repo has to be updated.

Reply:
Ack..issue with my local repo i guess. I think I got into master and pushed rebased from there. :p. Will redo this with main.